### PR TITLE
Updated Fonts

### PR DESCRIPTION
**Pull request by danieljfgoncalves**
Updated Fonts on LTOView @vincentrenais

### CODE REVIEW

**Comment by vincentrenais** [comment]
Awesome men, you actually understand Git, attaboy!

File: LTOView.m [hunk 1]
```diff
@@ -14,19 +15,18 @@ -(instancetype)initWithFrame:(CGRect)frame wasPrice:(NSNumber *)wasPrice savePri
     self = [super initWithFrame:frame];
     if (self) {
         
-#warning setup custom fonts
         // save price label
         UILabel *save = [[UILabel alloc]initWithFrame:CGRectMake(0, 0, frame.size.width, frame.size.height / 2)];
         save.textAlignment = NSTextAlignmentRight;
         save.textColor = [UIColor redColor];
-        save.font = [UIFont fontWithName:@"TimesNewRomanPSMT" size:14.0f];
```
Comment (vincentrenais) [inline LTOView.m]: That sucks!

Comment (vincentrenais) [inline LTOView.m]: TimesNewRoman is a crappy font for screens

**Comment by danieljfgoncalves** [comment]
@vincentrenais checkout

**Comment by vincentrenais** [comment]
:shipit:

**Pull request merged**